### ISSUE TITLE
Role BUTTON is not announced with Virtual PC Cursor inside container with aria-roledescription

### ISSUE THREAD

**Issue by dobrinyonkov** [FreedomScientific/standards-support#510]
### Summary

In Virtual PC Cursor mode, role "button" is not announced for button elements inside a container with role "toolbar" and aria-roledescription. Instead the aria-roledescription value is announced when navigating the buttons.

Example:
1. Go to https://jsbin.com/fujeheruba/edit?html,output
2. Turn on JAWS and Virtual PC Cursor mode ON
4. Press UP/DOWN arrows to navigate the buttons.

### Expected result

 Role "button" to be announced

### Actual result

aria-roledescription value is announced instead of role "button", every time a button receives the "selection"

### Example

https://jsbin.com/fujeheruba/edit?html,output

#### JAWS version and build number
2021.2011.16 ILM
 
#### Operating System and version

Microsoft Windows 10 Enterprise
Version 10.0.18363 Build 18363

#### Browser and version:
Version 89.0.4389.82 (Official Build) (x86_64)

**Comment by JAWS-test** [comment]
Duplicate of https://github.com/FreedomScientific/VFO-standards-support/issues/288 and https://github.com/FreedomScientific/VFO-standards-support/issues/407

**Comment by StephenBeckettTPGi** [comment]
Can confirm the following behaviour with JAWS 2021.2102.34 ILM, Windows 10 Enterprise for following markup

```
  <div role="toolbar" aria-roledescription="Overflow">
    <button>Enabled</button>
    <button>Enabled</button>
    <button disabled>Disabled</button>
  </div>
```

Tabbing first button: JAWS announces "Overflow, enabled button"
Using virtual cursor to move focus to first button: "Enabled overflow"

For,
- Microsoft Edge Version 89.0.774.57 (Official build) (64-bit)
- Google Chrome Version 89.0.4389.90 (Official Build) (64-bit)
- Firefox 86.0.1 (64-bit)

**Comment by BenKeyFSI** [comment]
Reported as Bug 122983 {(ST510) Odd behavior of SayLine when buttons are inside a container with aria-roledescription}.

**Comment by dobrinyonkov** [comment]
Hello, just adding more research on this topic.

The heading levels of titles when navigating in Virtual PC cursor mode are also not announced

```html
  <div role="toolbar" aria-roledescription="Overflow">
    <h2>My app title</h2>
    <button>Enabled</button>
    <button disabled>Disabled</button>
  </div>

**Comment by JAWS-test** [comment]
The problem continues to occur (current Chrome, JAWS 2024). In my opinion, the problem is very serious.
